index.js

```diff
@@ -248,3 +250,37 @@ function generateImage (opt, callback) {
     callback(null, container);
   });
 }
+
+function RoundAllValue (obj, decimal = 0) {
+  for (var key in obj) {
+    if (obj.hasOwnProperty(key)) {
+      if (obj[key] !== null) {
+        if (typeof(obj[key]) == "object") {
+          RoundAllValue (obj[key], decimal);
+        } else {
+          if(isNumeric(obj[key])) {
+            var oldNumber = obj[key];
+            obj[key] = roundNumber(obj[key], decimal);
+          }
+        }
+      }
+    }
+  }
+}
+
+function isNumeric (n) {
+  return !isNaN(parseFloat(n)) && isFinite(n);
```

Comment:
This is based on this solution [Validate decimal numbers in JavaScript](https://stackoverflow.com/questions/18082/validate-decimal-numbers-in-javascript-isnumeric).
I'm new to js, not sure whether all nodejs version use ES6. If so we could use Number.isFinite() instead?